include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32                 layeredShadingRateAttachments;
+    //VkBool32                 fragmentShadingRateNonTrivialCombinerOps;
+    //VkExtent2D               maxFragmentSize;
+    //uint32_t                 maxFragmentSizeAspectRatio;
+    //uint32_t                 maxFragmentShadingRateCoverageSamples;
+    //VkSampleCountFlagBits    maxFragmentShadingRateRasterizationSamples;
+    //VkBool32                 fragmentShadingRateWithShaderDepthStencilWrites;
+    //VkBool32                 fragmentShadingRateWithSampleMask;
+    //VkBool32                 fragmentShadingRateWithShaderSampleMask;
+    //VkBool32                 fragmentShadingRateWithConservativeRasterization;
+    //VkBool32                 fragmentShadingRateWithFragmentShaderInterlock;
+    //VkBool32                 fragmentShadingRateWithCustomSampleLocations;
+    //VkBool32                 fragmentShadingRateStrictMultiplyCombiner;
+    
+    /* Maintenance2PropertiesKHR *//* provided by VK_KHR_maintenance2 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance3PropertiesKHR *//* provided by VK_KHR_maintenance3 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance4PropertiesKHR *//* provided by VK_KHR_maintenance4 *//* MOVED TO Vulkan 1.3 Core  */
+    /* MultiviewPropertiesKHR    *//* provided by VK_KHR_multiview    *//* MOVED TO Vulkan 1.1 Core  */
+
+    /* PerformanceQueryPropertiesKHR *//* provided by VK_KHR_performance_query */
+    // ! We don't support PerformanceQueries at the moment;
+    // ! But we have a bool with the same name in SFeatures and that is mostly for GL when NBL_ARB_query_buffer_object is reported and that holds for every query 
+    // VkBool32           allowCommandBufferQueryCopies;
+
+    /* VK_KHR_portability_subset - PROVISINAL/NOTAVAILABLEANYMORE */
+
+    /* PushDescriptorPropertiesKHR *//* provided by VK_KHR_push_descriptor */
+    //uint32_t           maxPushDescriptors;
+
+    /* RayTracingPipelinePropertiesKHR *//* provided by VK_KHR_ray_tracing_pipeline */
+    uint32_t           shaderGroupHandleSize;
+    uint32_t           maxRayRecursionDepth;
+    uint32_t           maxShaderGroupStride;
+    uint32_t           shaderGroupBaseAlignment;
+    uint32_t           shaderGroupHandleCaptureReplaySize;
+    uint32_t           maxRayDispatchInvocationCount;
+    uint32_t           shaderGroupHandleAlignment;
+    uint32_t           maxRayHitAttributeSize;
+
+    /* FloatControlsPropertiesKHR *//* VK_KHR_shader_float_controls *//* MOVED TO Vulkan 1.2 Core  */
+    /* ShaderIntegerDotProductFeaturesKHR *//* VK_KHR_shader_integer_dot_product *//* MOVED TO Vulkan 1.3 Core  */
+    /* TimelineSemaphorePropertiesKHR *//* VK_KHR_timeline_semaphore *//* MOVED TO Vulkan 1.2 Core  */
+    /* VK_KHX_multiview *//* replaced by VK_KHR_multiview */
+
+    /* MultiviewPerViewAttributesPropertiesNVX *//* VK_NVX_multiview_per_view_attributes */
+    // VkBool32           perViewPositionAllComponents;
```

Comment:
https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceMultiviewPerViewAttributesPropertiesNVX.html

KHX/NVX means beta/experimental, lets not expose any of those